src/Formatter/FixSpaces.php

```diff
@@ -20,20 +20,19 @@ final class FixSpaces
 		'([§\*†©])\s' => '$1&nbsp;',
 	];
 
-
 	/**
 	 * @param string $haystack
 	 * @return string
 	 */
-	public function fix(string $haystack): string
+	public static function fix(string $haystack): string
 	{
-		$haystack = (string) preg_replace('/(\&nbsp\;|\s)+/', ' ', $haystack);
+		$haystack = (string)preg_replace('/(\&nbsp\;|\s)+/', ' ', $haystack);
```

Comment:
@miloshavlicek I think space should be here.

Reply:
I'll fix it later today.

Reply:
Is there any simple way how to run the fixer before commit?

Reply:
I am using this configuration and PhpStorm codestyle custom rules:

```
# Install packages
- php composer.phar create-project nette/code-checker temp/code-checker ^3 --no-progress
- php composer.phar create-project nette/coding-standard temp/coding-standard ^2 --no-progress

# Run test
- php temp/code-checker/code-checker --short-arrays --strict-types --fix --no-progress
- php temp/coding-standard/ecs check src --config temp/coding-standard/coding-standard-php71.yml
```

Thanks.